packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7773,122 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('multitag should have same set of attributes like in simple tags', function(assert) {
+        const sampleData = ['First Item', 'Second Item', 'Third Item', 'Fourth Item'];
+        const $tagBox = $('#tagBox').dxTagBox({
+            items: sampleData,
+            value: sampleData,
+            maxDisplayedTags: 2,
+            showMultiTagOnly: false,
+        });
+        const $simpleTagContainer = $tagBox.find(`.${TAGBOX_TAG_CLASS}:not(${TAGBOX_MULTI_TAG_CLASS})`);
+        const $multiTagContainer = $tagBox.find(`.${TAGBOX_MULTI_TAG_CLASS}`);
+        const simpleTagAttributes = Array.from($simpleTagContainer.prop('attributes')).map(attr => attr.name);
+        const multiTagAttributes = Array.from($multiTagContainer.prop('attributes')).map(attr => attr.name);
+
+        assert.strictEqual(multiTagAttributes.every(item => simpleTagAttributes.includes(item)), true, 'multitags has same attributes as simpletags');
```

Comment:
You need to check that the multitag has all the right attributes (with correct values), instead of comparing the attributes of two tags.

Reply:
[f086478](https://github.com/DevExpress/DevExtreme/pull/27063/commits/f086478a89c25c1071298b9fdace06ddbf2db9aa)